cinema_service/settings.py

```diff
@@ -43,6 +43,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "rest_framework",
+    'django_filters',
```

Comment:
Code Style: Consider using parentheses to group imports for better readability, as suggested in the checklist.